ArchiSteamFarm/Bot.cs

```diff
@@ -3009,7 +3009,7 @@ public sealed class Bot : IAsyncDisposable {
 
 				try {
 					inventory = await ArchiWebHandler.GetInventoryAsync(SteamID)
-						.Where(item => item.Tradable && (BotConfig.SetTypesToComplete.Contains(item.Type) && appIDs.Contains(item.RealAppID)))
+						.Where(item => item.Tradable && (BotConfig.CompleteTypesToSend.Contains(item.Type) && appIDs.Contains(item.RealAppID)))
```

Comment:
In general in situations like that I tend up to start with local variables and then move to global ones. It's irrelevant in the big picture, but it makes sense to check `appIDs` prior to `BotConfig.CompleteTypesToSend`.

Reply:
Memory locality is the only reason why you'd ever want to consider doing that instead of the other way. And in 99.999% cases you shouldn't bother with it, but when you're writing code you're already spending time on the line, so you can as well try to make it better.

Reply:
Now that's what I call nitpicky